integrated_channels/sap_success_factors/migrations/0014_drop_historical_table.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import connection, migrations
+
+
+def dropHistoricalTable(apps, schema_editor):
```

Comment:
i'm wondering if you need to remove the historical records here too? https://github.com/edx/edx-enterprise/blob/master/integrated_channels/sap_success_factors/models.py#L101

also i'm curious about what the PII is in this table?

Reply:
You're right. I should remove that line. I think the PII in this table is the success factors user ID.

Reply:
I agree, sorry I missed that.

Reply:
and just to double check, would running makemigrations now create the migration to delete the table without having to do the custom migration to delete it?

Reply:
I'm not sure, because I can't actually get edx-enterprise up and running locally. `make requirements` fails consistently. I'll give it a bit more of a shot before I merge this though.

Reply:
hmm, feel free to drop by if you are still having trouble. i'm on the 10th floor now, happy to help!